andela/AndelaEatsUI
# #164957160 Admin - Meal Sessions UI

#### What does this PR do?
- It creates the UI for meal sessions actions, ie, CRUD.

#### Description of Task to be completed?
- An admin user should be able to create, update, delete and view meal sessions.

#### How should this be manually tested?
- Clone the repo and checkout the `meal-sessions-ui-164957160` branch.
- Run the application and navigate to the Admin dashboard.
- Click on the `Menus` tab in the side navigation.
- You should see a third tab, `Meal Sessions` in addition to the `Meals` and `Menus` tabs.
- Click the `Meal Sessions` tab.
- You should see the list of meal sessions and the __Create__, __Update__ and __Delete__ buttons
#### What are the relevant pivotal tracker stories?
- [#164957160](https://www.pivotaltracker.com/story/show/164957160)

#### Background Context
- None.

#### Screenshot
<img width="1431" alt="Screen Shot 2019-04-18 at 10 05 05" src="https://user-images.githubusercontent.com/30738053/56342737-86395f00-61c1-11e9-81a8-c92e65c2843e.png">
